### DISCUSSION TITLE
trackingjs experimentation

### DISCUSSION THREAD

**Discussion by marcusx2** [AR-js-org/AR.js#307] [Ideas]
[https://github.com/eduardolundgren/tracking.js/issues/389](https://github.com/eduardolundgren/tracking.js/issues/389)

Maybe trackingjs can be used to replace or improve ARJS' current tracking?

**Comment by nickw1** [comment]
Did some experiments with trackingjs and found it was effective for locating feature points.

However we also need some way of converting 2D feature coordinates to 3D world coordinates. Have been in discussion with @ThorstenBux and @kalwalt over this and the way forward is perhaps PTAM (see https://github.com/nickw1/ptam_expts)
